src/main/java/uk/gov/companieshouse/authcodenotification/config/InterceptorConfig.java

```diff
@@ -12,8 +12,8 @@
 @ComponentScan("uk.gov.companieshouse.api.interceptor")
 public class InterceptorConfig implements WebMvcConfigurer {
 
-    static final String ALL_PATHS = "/**";
-    static final String HEALTH_CHECK_PATH = "/auth-code-notification/healthcheck";
+    private static final String ALL_PATHS = "/**";
+    private static final String HEALTH_CHECK_PATH = "/**/healthcheck";
```

Comment:
Here I think we need to think about this a little more. The interceptor excludes paths that match this right? So these paths don't require checks. Is there anything strange that can happen with the reg ex? E.g. if we include /../.. or similar, these would get through the interceptor. Can we construct a valid url request that then wouldn't be checked? Might be worth a test.

Reply:
i'll try various combinations and see what happens

Reply:
realised that the health check path is defined in the /resources/application.properties file which properties can be injected from, so I've injected the path from there to allow us to specify an explicit url path to exclude from the internal user checks.